tests/functional/s3cmd/tests.js

```diff
@@ -296,9 +310,13 @@ describe('s3cmd put and get bucket ACLs', function aclBuck() {
         `--acl-grant=write:${emailAccount}`], done);
     });
 
-    it('should get specific ACL that was set', done => {
+    // TODO: Unskip when getBucketLocation has been implemented
```

Comment:
@LaurenSpiegel @electrachong Do you want to merge getBucketLocation implementation: https://github.com/scality/S3/pull/535 first?